docs/hoon/library/1c.md

```diff
@@ -9,243 +9,295 @@ navhome: /docs/
 
 Blocksize
 
-Atom representing a blocksize, by convention expressed as a power of 2.
+Atom representing block size. A block of size `a` has a bitwidth of `2^a`.
 
-Source
-------
+#### Source
 
-    ++  bloq  @                                            ::  blockclass
+    ++  bloq  @                                         
 
-Examples
---------
+#### Examples
 
-    > :: ++met measures how many bloqs long an atom is
-    > (met 3 256)
+    ::++met measures how many a-bloqs long an atom is. In other words, how
+    many bloqs of size a would be needed to represent b.
+
+    > (met 3 256)                      ::one 3-bloq has a bidwidth of 8, which can represent 0-255
+    2
+
+    > (met 3 255)                      
+    1
+
+    > (met 3 65.535)                   ::two 3-bloqs together have a bidwidth of 16, which can represent 0-65,535                  
+    > 2              
+
```

Comment:
Some examples have a `>` before the return value and some don't.  You should remove these because they aren't consistent with what people see in the dojo.